src/pages/settings/Preferences/PriorityModePage.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useCallback} from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as User from '@userActions/User';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type PriorityModeItem = {
+    value: ValueOf<typeof CONST.PRIORITY_MODE>;
+    text: string;
+    alternateText: string;
+    keyForList: ValueOf<typeof CONST.PRIORITY_MODE>;
+    isSelected: boolean;
+};
+
+type PriorityModePageOnyxProps = {
+    /** The chat priority mode */
+    priorityMode: OnyxEntry<ValueOf<typeof CONST.PRIORITY_MODE>>;
```

Comment:
What do you think to move this to a separate type in `src/types/onyx` and reuse here?

Reply:
Do you mean `PriorityModePageOnyxProps` ?

Reply:
I meant to declare a type like this:
```ts
type PriorityMode = OnyxEntry<ValueOf<typeof CONST.PRIORITY_MODE>>;
```

And reuse in the codebase, what do you think?

Reply:
Yes, let's do both!